examples/text-generation/run_generation.py

```diff
@@ -227,6 +227,21 @@ def setup_parser(parser):
         action="store_true",
         help="Whether to enable Habana Flash Attention, provided that the model supports it.",
     )
+    parser.add_argument(
```

Comment:
This seems as a counter intuitive argument for inferencing.

Reply:
If there is no performance penalty and memory is also saved then we can internally pass it as True for 1st token when flash attention is used.

Reply:
Discussed offline. Here is summary:
We should left it controllable, cause it may be reasonable to turn it off for example on finetuning.
Additionnaly this parameter may cause some slight overhead even on inference, so maybe it's reasonable to turn it off if we don't need it.